book/data_sources.qmd

```diff
@@ -1,37 +1,32 @@
+# Data sources
 
-# Data Sources
+This page tabulates and provides links to all data sources used in this project.
+For more details on how each dataset is used and processed, refer to the [Methodology](method.qmd) section.
 
-The list of data sources used in this project. For more details on how is each dataset
-used and processed, refer to the [Methodology](method.qmd).
-
-## List of data source
-
-Below is a list of data sources organised by the purpose
-
-### Project-wide
+## Project-wide
 
 - OS | 2011 Census Output Areas | [link](https://geoportal.statistics.gov.uk/datasets/ons::output-areas-dec-2011-boundaries-ew-bgc/about)
 
-### Accessibility indicators
+## Accessibility indicators
 
 - ONS | Population Weighted Centroids (PWC) of Output Areas | [link](https://www.ons.gov.uk/census/2001censusandearlier/dataandproducts/outputgeography/outputareas)
 - ONS | Workplace zones | [link](https://www.nomisweb.co.uk/sources/census_2011_wp)
 - OS | Open Greenspace  | [link](https://beta.ordnancesurvey.co.uk/products/os-open-greenspace)
 - OSM | OpenStreetMap network data | [link](https://openstreetmap.org)
 - ITS Leeds | UK GTFS data | [link](https://itsleeds.github.io/UK2GTFS/)
```

Comment:
The UK GTFS data links to an R package which (I'm guessing) is used to generate the data. I also see that they have example data uploaded as GitHub releases.

However, it's not immediately clear to me exactly which dataset was being used. Is it [this one](https://github.com/ITSLeeds/UK2GTFS/releases/tag/0.003)?

Reply:
@ciupava ?

Reply:
https://data.bus-data.dft.gov.uk/downloads/

Reply:
timetable data